crypto/rsa/rsa_sign.c

```diff
@@ -116,9 +116,9 @@ static const unsigned char digestinfo_mdc2_der[] = {
 # ifndef OPENSSL_NO_RMD160
 /* RIPEMD160 (1 3 36 3 3 1 2) */
 static const unsigned char digestinfo_ripemd160_der[] = {
-    ASN1_SEQUENCE, 0x0c + RIPEMD160_DIGEST_LENGTH,
+    ASN1_SEQUENCE, 0x0e + RIPEMD160_DIGEST_LENGTH,
       ASN1_SEQUENCE, 0x08,
```

Comment:
This still is not right..

Reply:
ASN1_SEQUENCE, 0x08, is incorrect I think..

Reply:
Fixed